src/components/SelectEvent.js

```diff
@@ -176,7 +173,7 @@ const SportSection = ({section, eventSelected}) => {
     )
 }
 
-const OutcomesRow = ({event, wagerMembers, selectedOutcome, selectedMarket, eventSelected, rowNum}) => {
+const OutcomesRow = ({event, wagerMembers, selectedOutcome, selectedMarket, eventSelected, competitorId}) => {
     const auth = useStoreState(state => state.firebase.auth);
     if (!!wagerMembers) {
         const selected = wagerMembers[rowNum]?.uid === auth.uid;
```

Comment:
@donniemattingly , how can rowNum refer both to the "wagerMembers" and the outcome?

Reply:
It's from src/components/PersonalWagers.js:236

```javascript

const membersFromWager = (wager) => {
    return {
        [wager.details.outcome]: wager.proposedBy,
        [(wager.details.outcome + 1) % 2]: wager.proposedTo
    }
}

```

Basically when I save a wager I saved the outcome selected by the person proposing it, and assume the 'other' outcome must be the person being proposed to.

Reply:
so this is only for displaying saved wagers?

Reply:
yeah, that's correct - also for 'confirming' wagers, but that screen doesn't show anymore